adal/src/main/java/com/microsoft/aad/adal/AuthorityValidationMetadataCache.java

```diff
@@ -69,7 +69,7 @@ static boolean isAuthorityValidated(final URL authorityUrl) {
         return containsAuthorityHost(authorityUrl) && getCachedInstanceDiscoveryMetadata(authorityUrl).isValidated();
     }
 
-    static InstanceDiscoveryMetadata getCachedInstanceDiscoveryMetadata(final URL authorityUrl) {
+    public static InstanceDiscoveryMetadata getCachedInstanceDiscoveryMetadata(final URL authorityUrl) {
```

Comment:
This class isn't `public` but these methods are.... 🤔

Reply:
these methods are used in Ad-accounts, but I double checked they're used in the same package "com.microsoft.aad.adal", so it's OK to remove the added 'public'